src/e2e-test/java/io/cdap/plugin/utils/PubSubClient.java

```diff
@@ -36,6 +68,22 @@ public static Topic createTopic(String topicId) throws IOException {
     }
   }
 
+  // Create the subscription
+  public static void createSubscription(String subscriptionId, String topicId) throws IOException {
```

Comment:
Add a proper java doc instead of a one liner comment.

Reply:
added a java doc.